### DISCUSSION TITLE
Is there a problem about following the backend tutorial?

### DISCUSSION THREAD

**Discussion by Meminseeker** [bounswe/bounswe2024group11#186] [General]
As the backend team, we decided to follow and implement a simple tutorial about the Django REST Framework. I previously shared [a guide](https://github.com/bounswe/bounswe2024group11/wiki/How-to-Prepare-Your-Local-PC-for-the-Backend-Development) for you to onboard to the project. 

How is it going? Is everything okay for you? Did you encounter any problems during implementation? If you have any issues, please share them so that we can support you.

@MucahitErdoganUnlu @ceydanursen @yunuskaydin @ozankrkya

**Comment by Meminseeker** [comment]
If you encounter a `no such table: django_session` problem, I know the reason. Fortunately, the solution is easy as it is one line of command. If you have this issue, please let me know.

**Comment by MucahitErdoganUnlu** [comment]
Some of them are not suitable for windows unfortunately

**Reply by MucahitErdoganUnlu** [reply]
I added the windows options for these commands

**Comment by MucahitErdoganUnlu** [comment]
I completed the implementation of the tutorial.
